pkg/generic/thrift/util.go

```diff
@@ -59,6 +59,9 @@ func requestMappingValue(ctx context.Context, req *descriptor.HTTPRequest, field
 	case json.Number:
 		return buildinTypeFromString(string(v), t)
 	case string:
+		if v == "" && t.Type == descriptor.I64 {
```

Comment:
From a universal design perspective, we should return v if v is an empty string whatever t.Type is. In this way, If we need to add a new annotation, we won't meet a similar error.